packages/quantic/force-app/examples/main/lwc/exampleQuanticResultQuickview/exampleQuanticResultQuickview.js

```diff
@@ -0,0 +1,67 @@
+import { registerComponentForInit } from 'c/quanticHeadlessLoader';
+import {api, LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticResultQuickview extends LightningElement {
+    @api engineId = 'quantic-result-quickview-engine';
+    @track config = {};
+    isConfigured = false;
+
+
+    connectedCallback() {
+        registerComponentForInit(this, this.engineId);
+    }
+
+    result = {
+        title: "Test",
+        hasHtmlVersion: true,
+        uniqueId: "5",
+        raw: {
+            date: 1591016242000,
+            urihash: "5"
+        },
+        uri: "https://google.com",
+        clickUri: "https://google.com"
+    }
+
+    pageTitle = 'Quantic Result Quickview';
+    pageDescription = 'The Quantic Result Quickview component renders a button which the end user can click to open a modal box containing certain information about a result.'
+    options = [
+        {
+            attribute: 'result',
+            label: 'Result',
+            description: 'The result to retrieve a quickview for.',
```

Comment:
The field currently show as empty even if the default result is set. This will be very confusing for a user trying to use the form.

I guess we should, at least, show the object as JSON in the field. It would allow someone to easily alter the object to test various cases.

Reply:
done